### PR TITLE
Check for invalid GroupAssignmentRepos

### LINKED ISSUE #1505: ActionView::Template::Error: undefined method `github_team' for nil:NilClass
**Describe the bug**

```
ActionView::Template::Error
undefined method `github_team' for nil:NilClass
```

<table>
  <thead>
    <tr>
      <th>attribute</th>
      <th>value</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>action</td>
      <td>show</td>
    </tr>
    <tr>
      <td>cause</td>
      <td><pre>NoMethodError: undefined method `github_team&#39; for nil:NilClass<br>	/app/app/models/group_assignment_repo.rb:52:in `github_team&#39;<br>	/app/app/models/group_assignment_repo.rb:44:in `disabled?&#39;<br>	/app/app/view_models/shared_assignment_repo_view.rb:11:in `disabled?&#39;<br>	/app/app/view_models/shared_assignment_repo_view.rb:39:in `disabled_class&#39;<br></pre></td>
    </tr>
    <tr>
      <td>class_name</td>
      <td>ActionView::Template::Error</td>
    </tr>
    <tr>
      <td>controller</td>
      <td>orgs/group_assignment_repos</td>
    </tr>
    <tr>
      <td>method</td>
      <td>GET</td>
    </tr>
    <tr>
      <td>params</td>
      <td>{}</td>
    </tr>
    <tr>
      <td>referrer</td>
      <td>https://classroom.github.com/classrooms/40213532-classroom-test-org-edon/group-assignments/ga-1</td>
    </tr>
    <tr>
      <td>ruby</td>
      <td>ruby 2.4.2p198 (2017-09-14 revision 59899) [x86_64-linux]</td>
    </tr>
    <tr>
      <td>session</td>
      <td>{}</td>
    </tr>
    <tr>
      <td>time</td>
      <td>0.209490492</td>
    </tr>
    <tr>
      <td>url</td>
      <td>https://classroom.github.com/classrooms/40213532-classroom-test-org-edon/group-assignments/ga-1/group_assignment_repos/83679</td>
    </tr>
    <tr>
      <td>user</td>
      <td>false</td>
    </tr>
    <tr>
      <td>user_agent</td>
      <td>Mozilla/5.0 (Macintosh; Intel Mac OS X 10_13_6) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/11.1.2 Safari/605.1.15</td>
    </tr>

  </tbody>
</table>

<details>
<summary>Backtrace</summary>
<pre>
<span class="l">/app/app/models/group_assignment_repo.rb:52:in `github_team&#39;</span>
<span class="l">/app/app/models/group_assignment_repo.rb:44:in `disabled?&#39;</span>
<span class="l">/app/app/view_models/shared_assignment_repo_view.rb:11:in `disabled?&#39;</span>
<span class="l">/app/app/view_models/shared_assignment_repo_view.rb:39:in `disabled_class&#39;</span>
<span class="l">/app/app/views/orgs/group_assignment_repos/show.html.erb:3:in `_app_views_orgs_group_assignment_repos_show_html_erb__4105816352489853513_42567000&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/actionview-5.1.6/lib/action_view/template.rb:157:in `block in render&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/activesupport-5.1.6/lib/active_support/notifications.rb:166:in `block in instrument&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/activesupport-5.1.6/lib/active_support/notifications/instrumenter.rb:21:in `instrument&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/activesupport-5.1.6/lib/active_support/notifications.rb:166:in `instrument&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/actionview-5.1.6/lib/action_view/template.rb:352:in `instrument_render_template&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/actionview-5.1.6/lib/action_view/template.rb:155:in `render&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/actionview-5.1.6/lib/action_view/renderer/template_renderer.rb:52:in `block (2 levels) in render_template&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/actionview-5.1.6/lib/action_view/renderer/abstract_renderer.rb:42:in `block in instrument&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/activesupport-5.1.6/lib/active_support/notifications.rb:166:in `block in instrument&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/activesupport-5.1.6/lib/active_support/notifications/instrumenter.rb:21:in `instrument&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/activesupport-5.1.6/lib/active_support/notifications.rb:166:in `instrument&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/actionview-5.1.6/lib/action_view/renderer/abstract_renderer.rb:41:in `instrument&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/actionview-5.1.6/lib/action_view/renderer/template_renderer.rb:51:in `block in render_template&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/actionview-5.1.6/lib/action_view/renderer/template_renderer.rb:59:in `render_with_layout&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/actionview-5.1.6/lib/action_view/renderer/template_renderer.rb:50:in `render_template&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/actionview-5.1.6/lib/action_view/renderer/template_renderer.rb:14:in `render&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/actionview-5.1.6/lib/action_view/renderer/renderer.rb:42:in `render_template&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/actionview-5.1.6/lib/action_view/renderer/renderer.rb:23:in `render&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/actionview-5.1.6/lib/action_view/rendering.rb:103:in `_render_template&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/actionpack-5.1.6/lib/action_controller/metal/streaming.rb:217:in `_render_template&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/actionview-5.1.6/lib/action_view/rendering.rb:83:in `render_to_body&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/actionpack-5.1.6/lib/action_controller/metal/rendering.rb:52:in `render_to_body&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/actionpack-5.1.6/lib/action_controller/metal/renderers.rb:141:in `render_to_body&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/actionpack-5.1.6/lib/abstract_controller/rendering.rb:24:in `render&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/actionpack-5.1.6/lib/action_controller/metal/rendering.rb:36:in `render&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/actionpack-5.1.6/lib/action_controller/metal/instrumentation.rb:44:in `block (2 levels) in render&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/activesupport-5.1.6/lib/active_support/core_ext/benchmark.rb:12:in `block in ms&#39;</span>
<span class="l">/app/vendor/ruby-2.4.2/lib/ruby/2.4.0/benchmark.rb:308:in `realtime&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/activesupport-5.1.6/lib/active_support/core_ext/benchmark.rb:12:in `ms&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/actionpack-5.1.6/lib/action_controller/metal/instrumentation.rb:44:in `block in render&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/actionpack-5.1.6/lib/action_controller/metal/instrumentation.rb:87:in `cleanup_view_runtime&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/activerecord-5.1.6/lib/active_record/railties/controller_runtime.rb:29:in `cleanup_view_runtime&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/actionpack-5.1.6/lib/action_controller/metal/instrumentation.rb:43:in `render&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/turbolinks-2.5.3.226.g37a7c29/lib/turbolinks/redirection.rb:39:in `render&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/actionpack-5.1.6/lib/action_controller/metal/implicit_render.rb:33:in `default_render&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/actionpack-5.1.6/lib/action_controller/metal/basic_implicit_render.rb:4:in `block in send_action&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/actionpack-5.1.6/lib/action_controller/metal/basic_implicit_render.rb:4:in `tap&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/actionpack-5.1.6/lib/action_controller/metal/basic_implicit_render.rb:4:in `send_action&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/actionpack-5.1.6/lib/abstract_controller/base.rb:186:in `process_action&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/actionpack-5.1.6/lib/action_controller/metal/rendering.rb:30:in `process_action&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/actionpack-5.1.6/lib/abstract_controller/callbacks.rb:20:in `block in process_action&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/activesupport-5.1.6/lib/active_support/callbacks.rb:131:in `run_callbacks&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/actionpack-5.1.6/lib/abstract_controller/callbacks.rb:19:in `process_action&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/actionpack-5.1.6/lib/action_controller/metal/rescue.rb:20:in `process_action&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/actionpack-5.1.6/lib/action_controller/metal/instrumentation.rb:32:in `block in process_action&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/activesupport-5.1.6/lib/active_support/notifications.rb:166:in `block in instrument&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/activesupport-5.1.6/lib/active_support/notifications/instrumenter.rb:21:in `instrument&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/activesupport-5.1.6/lib/active_support/notifications.rb:166:in `instrument&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/actionpack-5.1.6/lib/action_controller/metal/instrumentation.rb:30:in `process_action&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/actionpack-5.1.6/lib/action_controller/metal/params_wrapper.rb:252:in `process_action&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/activerecord-5.1.6/lib/active_record/railties/controller_runtime.rb:22:in `process_action&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/actionpack-5.1.6/lib/abstract_controller/base.rb:124:in `process&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/actionview-5.1.6/lib/action_view/rendering.rb:30:in `process&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/actionpack-5.1.6/lib/action_controller/metal.rb:189:in `dispatch&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/actionpack-5.1.6/lib/action_controller/metal.rb:253:in `dispatch&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/actionpack-5.1.6/lib/action_dispatch/routing/route_set.rb:49:in `dispatch&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/actionpack-5.1.6/lib/action_dispatch/routing/route_set.rb:31:in `serve&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/actionpack-5.1.6/lib/action_dispatch/journey/router.rb:50:in `block in serve&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/actionpack-5.1.6/lib/action_dispatch/journey/router.rb:33:in `each&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/actionpack-5.1.6/lib/action_dispatch/journey/router.rb:33:in `serve&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/actionpack-5.1.6/lib/action_dispatch/routing/route_set.rb:844:in `call&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/newrelic_rpm-4.4.0.336/lib/new_relic/agent/instrumentation/middleware_tracing.rb:92:in `call&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/peek-performance_bar-1.2.0/lib/peek/views/performance_bar/process_utilization.rb:144:in `call&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/newrelic_rpm-4.4.0.336/lib/new_relic/agent/instrumentation/middleware_tracing.rb:92:in `call&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/omniauth-1.6.1/lib/omniauth/strategy.rb:189:in `call!&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/omniauth-1.6.1/lib/omniauth/strategy.rb:167:in `call&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/newrelic_rpm-4.4.0.336/lib/new_relic/agent/instrumentation/middleware_tracing.rb:92:in `call&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/omniauth-1.6.1/lib/omniauth/builder.rb:63:in `call&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/newrelic_rpm-4.4.0.336/lib/new_relic/agent/instrumentation/middleware_tracing.rb:92:in `call&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/newrelic_rpm-4.4.0.336/lib/new_relic/rack/agent_hooks.rb:30:in `traced_call&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/newrelic_rpm-4.4.0.336/lib/new_relic/agent/instrumentation/middleware_tracing.rb:92:in `call&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/newrelic_rpm-4.4.0.336/lib/new_relic/rack/browser_monitoring.rb:32:in `traced_call&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/newrelic_rpm-4.4.0.336/lib/new_relic/agent/instrumentation/middleware_tracing.rb:92:in `call&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/rack-tracker-1.4.0/lib/rack/tracker.rb:36:in `call&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/newrelic_rpm-4.4.0.336/lib/new_relic/agent/instrumentation/middleware_tracing.rb:92:in `call&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/pinglish-0.2.1/lib/pinglish.rb:42:in `call&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/newrelic_rpm-4.4.0.336/lib/new_relic/agent/instrumentation/middleware_tracing.rb:92:in `call&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/rack-2.0.5/lib/rack/etag.rb:25:in `call&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/newrelic_rpm-4.4.0.336/lib/new_relic/agent/instrumentation/middleware_tracing.rb:92:in `call&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/rack-2.0.5/lib/rack/conditional_get.rb:25:in `call&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/newrelic_rpm-4.4.0.336/lib/new_relic/agent/instrumentation/middleware_tracing.rb:92:in `call&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/rack-2.0.5/lib/rack/head.rb:12:in `call&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/newrelic_rpm-4.4.0.336/lib/new_relic/agent/instrumentation/middleware_tracing.rb:92:in `call&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/rack-2.0.5/lib/rack/session/abstract/id.rb:232:in `context&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/rack-2.0.5/lib/rack/session/abstract/id.rb:226:in `call&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/newrelic_rpm-4.4.0.336/lib/new_relic/agent/instrumentation/middleware_tracing.rb:92:in `call&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/actionpack-5.1.6/lib/action_dispatch/middleware/cookies.rb:613:in `call&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/newrelic_rpm-4.4.0.336/lib/new_relic/agent/instrumentation/middleware_tracing.rb:92:in `call&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/actionpack-5.1.6/lib/action_dispatch/middleware/callbacks.rb:26:in `block in call&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/activesupport-5.1.6/lib/active_support/callbacks.rb:97:in `run_callbacks&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/actionpack-5.1.6/lib/action_dispatch/middleware/callbacks.rb:24:in `call&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/newrelic_rpm-4.4.0.336/lib/new_relic/agent/instrumentation/middleware_tracing.rb:92:in `call&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/failbot-2.0.1/lib/failbot/middleware.rb:16:in `block in call&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/failbot-2.0.1/lib/failbot.rb:113:in `push&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/failbot-2.0.1/lib/failbot/middleware.rb:13:in `call&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/newrelic_rpm-4.4.0.336/lib/new_relic/agent/instrumentation/middleware_tracing.rb:92:in `call&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/airbrake-6.2.1/lib/airbrake/rack/middleware.rb:52:in `call&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/newrelic_rpm-4.4.0.336/lib/new_relic/agent/instrumentation/middleware_tracing.rb:92:in `call&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/actionpack-5.1.6/lib/action_dispatch/middleware/debug_exceptions.rb:59:in `call&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/newrelic_rpm-4.4.0.336/lib/new_relic/agent/instrumentation/middleware_tracing.rb:92:in `call&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/actionpack-5.1.6/lib/action_dispatch/middleware/show_exceptions.rb:31:in `call&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/newrelic_rpm-4.4.0.336/lib/new_relic/agent/instrumentation/middleware_tracing.rb:92:in `call&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/chewy-0.10.1/lib/chewy/railtie.rb:19:in `block in call&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/chewy-0.10.1/lib/chewy/strategy.rb:70:in `wrap&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/chewy-0.10.1/lib/chewy.rb:200:in `strategy&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/chewy-0.10.1/lib/chewy/railtie.rb:19:in `call&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/newrelic_rpm-4.4.0.336/lib/new_relic/agent/instrumentation/middleware_tracing.rb:92:in `call&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/lograge-0.6.0/lib/lograge/rails_ext/rack/logger.rb:15:in `call_app&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/railties-5.1.6/lib/rails/rack/logger.rb:24:in `block in call&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/activesupport-5.1.6/lib/active_support/tagged_logging.rb:69:in `block in tagged&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/activesupport-5.1.6/lib/active_support/tagged_logging.rb:26:in `tagged&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/activesupport-5.1.6/lib/active_support/tagged_logging.rb:69:in `tagged&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/railties-5.1.6/lib/rails/rack/logger.rb:24:in `call&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/newrelic_rpm-4.4.0.336/lib/new_relic/agent/instrumentation/middleware_tracing.rb:92:in `call&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/actionpack-5.1.6/lib/action_dispatch/middleware/remote_ip.rb:79:in `call&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/newrelic_rpm-4.4.0.336/lib/new_relic/agent/instrumentation/middleware_tracing.rb:92:in `call&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/request_store-1.3.2/lib/request_store/middleware.rb:9:in `call&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/newrelic_rpm-4.4.0.336/lib/new_relic/agent/instrumentation/middleware_tracing.rb:92:in `call&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/actionpack-5.1.6/lib/action_dispatch/middleware/request_id.rb:25:in `call&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/newrelic_rpm-4.4.0.336/lib/new_relic/agent/instrumentation/middleware_tracing.rb:92:in `call&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/rack-2.0.5/lib/rack/method_override.rb:22:in `call&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/newrelic_rpm-4.4.0.336/lib/new_relic/agent/instrumentation/middleware_tracing.rb:92:in `call&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/rack-2.0.5/lib/rack/runtime.rb:22:in `call&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/newrelic_rpm-4.4.0.336/lib/new_relic/agent/instrumentation/middleware_tracing.rb:92:in `call&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/activesupport-5.1.6/lib/active_support/cache/strategy/local_cache_middleware.rb:27:in `call&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/newrelic_rpm-4.4.0.336/lib/new_relic/agent/instrumentation/middleware_tracing.rb:92:in `call&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/actionpack-5.1.6/lib/action_dispatch/middleware/executor.rb:12:in `call&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/newrelic_rpm-4.4.0.336/lib/new_relic/agent/instrumentation/middleware_tracing.rb:92:in `call&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/actionpack-5.1.6/lib/action_dispatch/middleware/static.rb:125:in `call&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/newrelic_rpm-4.4.0.336/lib/new_relic/agent/instrumentation/middleware_tracing.rb:92:in `call&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/rack-2.0.5/lib/rack/sendfile.rb:111:in `call&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/newrelic_rpm-4.4.0.336/lib/new_relic/agent/instrumentation/middleware_tracing.rb:92:in `call&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/actionpack-5.1.6/lib/action_dispatch/middleware/ssl.rb:68:in `call&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/newrelic_rpm-4.4.0.336/lib/new_relic/agent/instrumentation/middleware_tracing.rb:92:in `call&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/railties-5.1.6/lib/rails/engine.rb:522:in `call&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/newrelic_rpm-4.4.0.336/lib/new_relic/agent/instrumentation/middleware_tracing.rb:92:in `call&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/rack-canonical-host-0.2.3/lib/rack/canonical_host.rb:19:in `call&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/newrelic_rpm-4.4.0.336/lib/new_relic/agent/instrumentation/middleware_tracing.rb:92:in `call&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/puma-3.10.0/lib/puma/configuration.rb:225:in `call&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/puma-3.10.0/lib/puma/server.rb:605:in `handle_request&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/puma-3.10.0/lib/puma/server.rb:437:in `process_client&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/puma-3.10.0/lib/puma/server.rb:301:in `block in run&#39;</span>
<span class="l">/app/vendor/bundle/ruby/2.4.0/gems/puma-3.10.0/lib/puma/thread_pool.rb:120:in `block in spawn_thread&#39;</span>
</pre>
</details>

**To Reproduce**
1. Go to `GroupAssignmentsController` where Groups have been deleted

**Expected behavior**
Expects to render the template

**Screenshots**
screen shot 2018-08-27 at 11 58 01 am

**Additional context**
(possibly related to the deletion of groups #1346 #868)

### PR DESCRIPTION
**Pull request by EricPickup**
## What

Fixes #1505.

#1346 was in production for a while and resulted in broken/invalid `GroupAssignmentRepo`s. These models had their associating `Group` model destroyed which eliminated the associating `github_team` for the repos.

This change makes us check for a nil `Group` and returns a `NullGitHubTeam` when requesting the repo's `github_team`, which will appear as a ghost team as shown in the screenshot below.

**Before:**

image

**After:**

image

### CODE REVIEW

File: app/models/group_assignment_repo.rb [hunk 1]
```diff
@@ -51,6 +51,8 @@ def github_repository
   end
 
   def github_team
+    return NullGitHubTeam.new if group.nil?
```
Comment (BenEmdon) [inline app/models/group_assignment_repo.rb]: Please write a test case for this 🙏

**Comment by d12** [comment]
Small note about `Fix for #1505`, if you word things a certain way, GitHub will close the issue you linked automatically when you merge the PR. You can tell if you did it right if there's a little squigly underline under the line. https://help.github.com/en/articles/closing-issues-using-keywords

```
Closes #1505
```

**Comment by EricPickup** [comment]
Thanks for the feedback @BenEmdon, tests have been added 🎉

**Comment by BenEmdon** [comment]
@EricPickup this branch can be safely deleted. ☝️ Are you reusing this branch?

**Pull request merged**